WeakAurasOptions/WeakAurasOptions.lua

```diff
@@ -3003,47 +3002,13 @@ function WeakAuras.ReloadTriggerOptions(data)
       end,
       order = 2
     },
-    typedesc = {
-      type = "toggle",
-      name = L["Type"],
-      order = 5,
-      disabled = true,
-      get = function() return true end
-    },
-    type = {
-      type = "select",
-      name = L["Type"],
-      desc = L["The type of trigger"],
-      order = 6,
-      values = trigger_types,
-      set = function(info, v)
-        trigger.type = v;
-        if(trigger.event) then
-          local prototype = WeakAuras.event_prototypes[trigger.event];
-          if(prototype) then
-            if(v == "status" and prototype.type == "event") then
-              trigger.event = "Health";
-            elseif(v == "event" and prototype.type == "status") then
-              trigger.event = "Chat Message";
-            end
-          end
-        end
-        WeakAuras.Add(data);
-        WeakAuras.SetThumbnail(data);
-        WeakAuras.SetIconNames(data);
-        WeakAuras.UpdateDisplayButton(data);
-        WeakAuras.ReloadTriggerOptions(data);
-      end
-    },
   };
 
   local function hideTriggerCombiner()
     return not (data.triggers.disjunctive == "custom")
   end
   WeakAuras.AddCodeOption(trigger_options, data, L["Custom"], "custom_trigger_combination", 0.1, hideTriggerCombiner, {"triggers", "customTriggerLogic"}, false);
 
-  trigger_options = union(trigger_options, WeakAuras.GetGenericTriggerOptions(data, trigger, untrigger));
```

Comment:
This adds the common generic trigger options and custom trigger options. This should only done for generic triggers, but needs a few more steps to reach the right place.